jobs/probe_scraper.sh

```diff
@@ -7,17 +7,25 @@ BUCKET="net-mozaws-prod-us-west-2-data-pitmo"
 CACHE_DIR="probe_cache"
 OUTPUT_DIR="probe_data"
 
+ANACONDA_PATH=/mnt/anaconda2
+ANACONDA_SCRIPT=Anaconda3-5.3.1-Linux-x86_64.sh
```

Comment:
You could probably get away with Miniconda for this, since we don't need large parts of the scientific stack (Scipy etc.)

Reply:
Good point. I'll test it on miniconda.